docker-compose.yml

```diff
@@ -743,6 +749,7 @@ services:
       DATABASE_URL: postgresql://postgres:postgres@postgres/content-tagger
       REDIS_URL: redis://redis
       SENTRY_CURRENT_ENV: content-tagger-worker
+      PORT: 3116
```

Comment:
If it does then that's all good. If not then I think you may need to avoid setting the extra healthcheck on the worker (as it's currently being inherited).

Reply:
will test this - i suspect your right